examples/delegator/build-all.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-pushd accumulator && cargo +nightly contract build --generate code-only && popd &&
-pushd adder && cargo +nightly contract build --generate code-only && popd &&
-pushd subber && cargo +nightly contract build --generate code-only && popd &&
+pushd accumulator && cargo +nightly contract build && popd &&
```

Comment:
You could use `--manifest-path=accumulator/Cargo.toml` instead of navigating in and out of directories?

Also this whole thing will be nicer with https://github.com/paritytech/cargo-contract/issues/182

Reply:
I think the script was written when `--manifest-path` has not yet been implemented. I agree we should use it or even better `-p`. Also we might profit from a `--workspace` flag to build the entire workspace of a root smart contract.

Reply:
>  I agree we should use it or even better -p. Also we might profit from a --workspace flag to build the entire workspace of a root smart contract

Indeed, see paritytech/cargo-contract#182